### PR TITLE
Add navigation

### PR DESCRIPTION
**Pull request by MxWoodie**
Add navigation with logo and underscore
Add comprehensive reset.css
Extend Gulpfile.js tracking coverage

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,19 +3,34 @@
 	<head>
 		<meta charset="UTF-8">
 		<title>Document</title>
-		<link rel="stylesheet" href="/assets/style.css" />
-	</head>
+		
+	<link rel="stylesheet" href="/assets/style.css" /></head>
```
Comment (ElizavetaSelivanova) [inline index.html]: Alignment

File: index.html [hunk 2]
```diff
@@ -3,19 +3,34 @@
...
-			<nav></nav>
+		<header class="page-header">
+			<nav class="navigation-block">
+				<ul class="navigation-container">
+					<li class="navigation-item-container"><a class="navigation-item selected" href="#">HOME</a></li>
+					<li class="navigation-item-container"><a class="navigation-item" href="#">PORTFOLIO</a></li>
+					<li class="navigation-item-container"><a class="navigation-item" href="#">BLOG</a></li>
+					<li class="navigation-item-container">
+						<a class="navigation-item navigation-item-logo" href="#">
+							<img class="logo" src="assets/img/logo.png" alt="expire-logo">
```
Comment (ElizavetaSelivanova) [inline index.html]: Close tag

File: index.html [hunk 3]
```diff
@@ -3,19 +3,34 @@
...
+	<link rel="stylesheet" href="/assets/style.css" /></head>
 	<body>
-		<header>
-			<nav></nav>
+		<header class="page-header">
+			<nav class="navigation-block">
+				<ul class="navigation-container">
+					<li class="navigation-item-container"><a class="navigation-item selected" href="#">HOME</a></li>
+					<li class="navigation-item-container"><a class="navigation-item" href="#">PORTFOLIO</a></li>
+					<li class="navigation-item-container"><a class="navigation-item" href="#">BLOG</a></li>
```
Comment (ElizavetaSelivanova) [inline index.html]: No need uppercase here - use styling for this (for all `<li>`)

File: index.html [hunk 4]
```diff
@@ -3,19 +3,34 @@
...
+					<li class="navigation-item-container">
+						<a class="navigation-item navigation-item-logo" href="#">
+							<img class="logo" src="assets/img/logo.png" alt="expire-logo">
+							<p class="title">Expire</p>
```
Comment (ElizavetaSelivanova) [inline index.html]: No need to use `<p>` here, and class `title` is not suitable for this

File: scss/components/_navigation.scss [hunk 5]
```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: "Raleway-Regular";
+  src: url(/assets/fonts/Raleway-Regular.ttf) format("truetype");
+  font-weight: 400;
+
```
Comment (ElizavetaSelivanova) [inline scss/components/_navigation.scss]: Remove indent

File: scss/components/_navigation.scss [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+@font-face {
+  font-family: "Raleway-SemiBold";
+  src: url(/assets/fonts/Raleway-SemiBold.ttf) format("truetype");
+  font-weight: 600;
+}
+
+.page-header {
+  background-color: #3D4148;
```
Comment (ElizavetaSelivanova) [inline scss/components/_navigation.scss]: Use variables

File: scss/components/_navigation.scss [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+  .navigation-item-container {
+    margin: 10px;
+    text-align: center;
+    
+    .navigation-item {
+      font-family: "Raleway-Light", "Raleway-Regular", sans-serif;
+      font-size: 18px;
+      text-decoration: none;
+      color: #FAFAFA;
```
Comment (ElizavetaSelivanova) [inline scss/components/_navigation.scss]: variable

File: scss/components/_navigation.scss [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+      &.selected {
+        position: relative;
+      }
+
+      &.selected::after {
+        content:'';
```
Comment (ElizavetaSelivanova) [inline scss/components/_navigation.scss]: Indent after `:`

File: scss/vendor/reset.css [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+	font-weight: inherit;
+	font-family: inherit;
+	font-style: inherit;
+	font-size: 100%;
+	border: 0 none;
+	outline: 0;
+	padding: 0;
+	margin: 0;
+	}
+
```
Comment (ElizavetaSelivanova) [inline scss/vendor/reset.css]: Alignment

File: index.html (hunk 2 above)
Comment (angelinabylitskaya) [inline index.html]: Tag img should be closed

File: index.html [hunk 10]
```diff
@@ -3,19 +3,34 @@
...
-	</head>
+		
+	<link rel="stylesheet" href="/assets/style.css" /></head>
 	<body>
-		<header>
-			<nav></nav>
+		<header class="page-header">
+			<nav class="navigation-block">
+				<ul class="navigation-container">
+					<li class="navigation-item-container"><a class="navigation-item selected" href="#">HOME</a></li>
```
Comment (angelinabylitskaya) [inline index.html]: Use css styles to uppercase text

File: scss/components/_navigation.scss [hunk 11]
```diff
@@ -0,0 +1,64 @@
+@font-face {
```
Comment (angelinabylitskaya) [inline scss/components/_navigation.scss]: Move fonts to `scss/_fonts.scss` file

File: scss/components/_navigation.scss (hunk 8 above)
Comment (angelinabylitskaya) [inline scss/components/_navigation.scss]: Use space after `:`

File: scss/components/_navigation.scss [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+      text-decoration: none;
+      color: #FAFAFA;
+
+      &.selected {
+        position: relative;
+      }
+
+      &.selected::after {
+        content:'';
+        width: 130%;
```
Comment (angelinabylitskaya) [inline scss/components/_navigation.scss]: Don't use %

File: scss/components/_navigation.scss [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+        position: absolute;
+        left: -15%;
+        bottom: -5px;
+        height: 1px;
+        background: linear-gradient(to right, #902616, #FE3C1F);
```
Comment (angelinabylitskaya) [inline scss/components/_navigation.scss]: Move colors to variables

File: scss/vendor/reset.css [hunk 14]
```diff
@@ -0,0 +1,12 @@
+* {
```
Comment (angelinabylitskaya) [inline scss/vendor/reset.css]: Update this file with
https://meyerweb.com/eric/tools/css/reset/

File: scss/components/_navigation.scss [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+  margin: 0 auto;
+  padding-top: 22px;
+  list-style-type: none;
+
+  .navigation-item-container {
+    margin: 10px;
+    text-align: center;
+    
+    .navigation-item {
+      font-family: "Raleway-Light", "Raleway-Regular", sans-serif;
```
Comment (angelinabylitskaya) [inline scss/components/_navigation.scss]: Move fonts mane to variable

File: scss/components/_navigation.scss (hunk 7 above)
Comment (angelinabylitskaya) [inline scss/components/_navigation.scss]: Use lower case for colors, move to variables

**Pull request merged**